exampleplugins/5.2/SwagSloganOfTheDay/Subscriber/RouteSubscriber.php

```diff
@@ -0,0 +1,45 @@
+<?php
+namespace SwagSloganOfTheDay\Subscriber;
+
+use Enlight\Event\SubscriberInterface;
+use Shopware\Components\Plugin\ConfigReader;
+use SwagSloganOfTheDay\Components\SloganPrinter;
+
+class RouteSubscriber implements SubscriberInterface
+{
+    private $pluginDirectory;
+    private $sloganPrinter;
+    private $config;
+
+    public static function getSubscribedEvents()
+    {
+        return [
+            'Enlight_Controller_Action_PostDispatchSecure_Frontend' => 'onPostDispatch'
+        ];
+    }
+
+    public function __construct($pluginName, $pluginDirectory, SloganPrinter $sloganPrinter, ConfigReader $configReader)
+    {
+        $this->pluginDirectory = $pluginDirectory;
+        $this->sloganPrinter = $sloganPrinter;
+
+        $this->config = $configReader->getByPluginName($pluginName);
+    }
+
+    public function onPostDispatch(\Enlight_Controller_ActionEventArgs $args)
+    {
+        /** @var \Enlight_Controller_Action $controller */
+        $controller = $args->get('subject');
+        $view = $controller->View();
+
+        $view->addTemplateDir($this->pluginDirectory . '/Resources/views');
+
+        $view->assign('fontSize', $this->config['fontSize']);
```

Comment:
please prefix the smarty assignments. fontSize, italic and slogan are too generic

Reply:
Done